dist/component-war/src/test/java/org/hawkular/bus/PublishDataPointsTest.java

```diff
@@ -130,6 +137,7 @@ public static void setupPublishing() {
         System.setProperty("hawkular-metrics.publish-period", "100");
     }
 
+    @Ignore
```

Comment:
I guess this test will be fixed.
The main scenario is to validate the filtering use case, having data points for published and non published metrics and checking that only the published are received in the old JMS.
In the new scenario, a mock AlertsServices as noted in the commented code just has to validate that points are send.